spec/features/user_searches_for_vendor.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'visitor searches for a vendor', %{
+  As a visitor
+  I want to search for a vendor by name
+  So that I can find my favorite vendor
+} do
+
+  scenario 'visitor searches for a vendor by name' do
```

Comment:
Change this scenario description to be more descriptive of what you're actually testing here!